Library/Formula/h2.rb

```diff
@@ -17,6 +17,9 @@ def install
     # Remove windows files
     rm_f Dir["bin/*.bat"]
 
+    system "cat bin/h2.sh | col -b > bin/_tmp_h2.sh"
```

Comment:
Doable, but will have to use a name other than `script` to prevent shadowing the one from a few lines above (`h2_script`?). No dependency on `col` would be a bonus, I guess?